cdap-docs/reference-manual/source/glossary.rst

```diff
@@ -134,5 +134,20 @@ Glossary
    Master Services
       CDAP system services that are run in YARN containers like Transaction Service,
       Dataset Executor, Log Saver, Metrics Processor, etc.
-  
+
+   Fileset
+      A :term:`Dataset` composed of collections of files in the file system that share
+      some common attributes such as the format and schema, which abstracts from the
+      actual underlying file system interfaces.
+
+   TimePartitioned Fileset
```

Comment:
Should be the same across the board - either Time-partitioned file set, or time-partitioned Fileset. Not sure. Typically the fragments of a camel-case word translate into separate words in text. Such as ApplicationSpecification becomes application specification. Or InputStream becomes input stream. Should we do the same? That would mean it is time partitioned file set.